src/components/NewMessage/NewMessagePollEditor.vue

```diff
@@ -12,19 +12,31 @@
 		<p class="poll-editor__caption">
 			{{ t('spreed', 'Question') }}
 		</p>
-		<NcTextField :value.sync="pollQuestion" :label="t('spreed', 'Ask a question')" v-on="$listeners" />
+		<NcTextField :value.sync="pollForm.question" :label="t('spreed', 'Ask a question')" v-on="$listeners" />
+		<!--native file picker, hidden -->
+		<input id="poll-upload"
+			ref="pollImport"
+			type="file"
+			class="hidden-visually"
+			@change="importPoll">
+		<NcButton class="poll-editor__button"
+			type="secondary"
+			wide
+			@click="triggerImport">
+			{{ t('spreed', 'Import poll from file') }}
+		</NcButton>
```

Comment:
Buttons locations are a bit blatant. What if we put them both in in a 3-dot menu on top? 
Also, exporting the poll should always be available (being created, active, closed)

Reply:
Next to the question looks also good enough?
<img width="352" alt="image" src="https://github.com/user-attachments/assets/d987a625-249a-4e5c-8a3b-a1be28293f9e">

Reply:
Export on other pages:
voting:
<img width="177" alt="image" src="https://github.com/user-attachments/assets/a7d36fa9-ac3a-46d2-883f-b5f48130a054">
results:
<img width="302" alt="image" src="https://github.com/user-attachments/assets/2e1a2502-c08e-48c5-9910-258269308668">

Reply:
> Next to the question looks also good enough?

How about a "Import" Next to the "Create" button
And only have the "Export" on already Created-polls

Reply:
I wouldn't expect 'Import' button to be close to the finish.
And personally, I can imagine cases where having 'Export' option before creation is useful:
- e.g. creating a poll beforehand in the same conversation (without posting) and save it locally, then restore from file during the call and post to people
- with same intention, but instead of locally saving keep it it Per-Conversation poll drafts (to post from there later)

Though these drafts would require some API work, especially if we want to prepare them during room creation, for example